docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,330 @@
+
+  - id: bucket-input-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_inputs)
+
+  - id: bucket-output-create
+    source: ./community/modules/file-system/cloud-storage-bucket
+    settings:
+      name_prefix: $(vars.bucket_name_outputs)
+
+- group: software_installation
+  modules:
+
+  - id: network-install  # This will reuse network created in setup group
+    source: modules/network/pre-existing-vpc
+
+  - id: appsfs-install
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: apps
+      local_mount: /apps
+      fs_type: nfs
+
+  - id: bucket-input-install
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_inputs)
+      local_mount: /data_input
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs,allow_other,dir_mode=0777,file_mode=766
+
+  ### Software ###
+
+  - id: spack
+    source: community/modules/scripts/spack-install
+    settings:
+      install_dir: /apps/spack
+      configs:
+      - type: file
+        scope: defaults
+        content: |
+          config:
+            build_stage:
+              - /apps/spack/spack-stage
+      - type: file
+        scope: defaults
+        content: |
+          modules:
+            default:
+              tcl:
+                hash_length: 0
+                all:
+                  conflict:
+                    - '{name}'
+                projections:
+                  all: '{name}/{version}-{compiler.name}-{compiler.version}'
+      - type: file
+        scope: site
+        content: |
+          packages:
+            slurm:
+              externals:
+                - spec: slurm@21-08-8-2
+                  prefix: /usr/local
+              buildable: False
+      compilers: [gcc@9.3.0 target=x86_64]
+      environments:
+      - name: gromacs
+        type: file
+        content: |
+          spack:
+            definitions:
+            - compilers:
+              - gcc@9.3.0
+            - mpis:
+              - intel-mpi@2018.4.274
+            - mpi_packages:
+              - gromacs@2020.5
+            specs:
+            - matrix:
+              - - $mpis
+              - - $%compilers
+            - matrix:
+              - - $mpi_packages
+              - - $%compilers
+              - - $^mpis
+
+  - id: spack-builder-startup
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(spack.install_spack_deps_runner)
+      - $(spack.install_spack_runner)
+      - type: shell
+        destination: data_staging.sh
+        content: |
+          #!/bin/bash
+          wget --no-verbose -P /data_input/protien_data_bank/ https://files.rcsb.org/download/1FJS.pdb
+          wget --no-verbose -P /data_input/gromacs_inputs https://ftp.gromacs.org/pub/benchmarks/water_GMX50_bare.tar.gz && \
```

Comment:
this is likely to go *significantly* faster if you wget to /tmp or even local PD and then extract to the gcs bucket.

Reply:
I have updated to do as you suggest.